account_invoice_split/wizard/account_invoice_split.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#     This file is part of account_invoice_split,
+#     an Odoo module.
+#
+#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+#
+#     account_invoice_split is free software:
+#     you can redistribute it and/or modify it under the terms of the GNU
+#     Affero General Public License as published by the Free Software
+#     Foundation,either version 3 of the License, or (at your option) any
+#     later version.
+#
+#     account_invoice_split is distributed
+#     in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+#     even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#     PURPOSE.  See the GNU Affero General Public License for more details.
+#
+#     You should have received a copy of the GNU Affero General Public License
+#     along with account_invoice_split.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api, exceptions, _
+import openerp.addons.decimal_precision as dp
+
+FIELD_MAPPING = [('id', 'origin_invoice_line_id'),
+                 ('product_id', 'product_id'),
+                 ('name', 'name'),
+                 ('quantity', 'total_quantity')]
```

Comment:
If you make it inmutable, then you can't extend it, which is the purpose I think